### ISSUE TITLE
 `Accelerate` default config: 	Not found

### ISSUE THREAD

**Issue by treksis** [brian6091/Dreambooth#46]
I get this error. 

This is my notebook. I'm check-marked LoRA and disabled the prior preservation option. Then the rests are in the default setting. 

https://colab.research.google.com/drive/1Kr4Fy1_wIhg_j7_rYLpDv40FLKnnDqen?usp=share_link

Thank you

```
No LSB modules are available.
Description:	Ubuntu 20.04.5 LTS
diffusers==0.11.1
lora-diffusion @ file:///content/lora
torchvision @ https://download.pytorch.org/whl/cu116/torchvision-0.14.1%2Bcu116-cp38-cp38-linux_x86_64.whl
transformers==4.25.1
xformers==0.0.16rc425

Copy-and-paste the text below in your GitHub issue

- `Accelerate` version: 0.15.0
- Platform: Linux-5.10.147+-x86_64-with-glibc2.29
- Python version: 3.8.10
- Numpy version: 1.21.6
- PyTorch version (GPU?): 1.13.1+cu116 (True)
- `Accelerate` default config:
	Not found
usage: accelerate <command> [<args>] launch
       [-h]
       [--config_file CONFIG_FILE]
       [--cpu]
       [--mps]
       [--multi_gpu]
       [--tpu]
       [--use_mps_device]
       [--dynamo_backend {no,eager,aot_eager,inductor,nvfuser,aot_nvfuser,aot_cudagraphs,ofi,fx2trt,onnxrt,ipex}]
       [--mixed_precision {no,fp16,bf16}]
       [--fp16]
       [--num_processes NUM_PROCESSES]
       [--num_machines NUM_MACHINES]
       [--num_cpu_threads_per_process NUM_CPU_THREADS_PER_PROCESS]
       [--use_deepspeed]
       [--use_fsdp]
       [--use_megatron_lm]
       [--gpu_ids GPU_IDS]
       [--same_network]
       [--machine_rank MACHINE_RANK]
       [--main_process_ip MAIN_PROCESS_IP]
       [--main_process_port MAIN_PROCESS_PORT]
       [--rdzv_conf RDZV_CONF]
       [--max_restarts MAX_RESTARTS]
       [--monitor_interval MONITOR_INTERVAL]
       [-m]
       [--no_python]
       [--main_training_function MAIN_TRAINING_FUNCTION]
       [--downcast_bf16]
       [--deepspeed_config_file DEEPSPEED_CONFIG_FILE]
       [--zero_stage ZERO_STAGE]
       [--offload_optimizer_device OFFLOAD_OPTIMIZER_DEVICE]
       [--offload_param_device OFFLOAD_PARAM_DEVICE]
       [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS]
       [--gradient_clipping GRADIENT_CLIPPING]
       [--zero3_init_flag ZERO3_INIT_FLAG]
       [--zero3_save_16bit_model ZERO3_SAVE_16BIT_MODEL]
       [--deepspeed_hostfile DEEPSPEED_HOSTFILE]
       [--deepspeed_exclusion_filter DEEPSPEED_EXCLUSION_FILTER]
       [--deepspeed_inclusion_filter DEEPSPEED_INCLUSION_FILTER]
       [--deepspeed_multinode_launcher DEEPSPEED_MULTINODE_LAUNCHER]
       [--fsdp_offload_params FSDP_OFFLOAD_PARAMS]
       [--fsdp_min_num_params FSDP_MIN_NUM_PARAMS]
       [--fsdp_sharding_strategy FSDP_SHARDING_STRATEGY]
       [--fsdp_auto_wrap_policy FSDP_AUTO_WRAP_POLICY]
       [--fsdp_transformer_layer_cls_to_wrap FSDP_TRANSFORMER_LAYER_CLS_TO_WRAP]
       [--fsdp_backward_prefetch_policy FSDP_BACKWARD_PREFETCH_POLICY]
       [--fsdp_state_dict_type FSDP_STATE_DICT_TYPE]
       [--megatron_lm_tp_degree MEGATRON_LM_TP_DEGREE]
       [--megatron_lm_pp_degree MEGATRON_LM_PP_DEGREE]
       [--megatron_lm_num_micro_batches MEGATRON_LM_NUM_MICRO_BATCHES]
       [--megatron_lm_sequence_parallelism MEGATRON_LM_SEQUENCE_PARALLELISM]
       [--megatron_lm_recompute_activations MEGATRON_LM_RECOMPUTE_ACTIVATIONS]
       [--megatron_lm_use_distributed_optimizer MEGATRON_LM_USE_DISTRIBUTED_OPTIMIZER]
       [--megatron_lm_gradient_clipping MEGATRON_LM_GRADIENT_CLIPPING]
       [--aws_access_key_id AWS_ACCESS_KEY_ID]
       [--aws_secret_access_key AWS_SECRET_ACCESS_KEY]
       [--debug]
       training_script
       ...
accelerate <command> [<args>] launch: error: argument --mixed_precision: invalid choice: '' (choose from 'no', 'fp16', 'bf16')
```

**Comment by brian6091** [comment]
@treksis Sometimes this error happens when there is a missing argument. Did you run all the cells in sequence?

**Comment by treksis** [comment]
Solved. By default, CONDITIONING_DROPOUT_PROB is set to 0 that was why.

Regarding to CONDITIONING_DROPOUT_PROB. It says "Probability that image (applies to both instance and class images) will be selected for dropout" Is this similar to when we train CNN, we purposely dropout to prevent overfitting? Like the higher number like 0.8 will prevent overfit because it drops more and then the lower number like 0.2, the vice versa?

thank you

**Comment by brian6091** [comment]
@treksis no it's not the same as dropout in a cnn layer. It refers to the probably of dropping the text conditioning in the diffusion process (the prompt). This will cause the prompt to be dropped for some fraction of steps, which has the effect of increasing fidelity to the training images, but at the cost of diversity of generated images. I haven't played with this much, but this was used to train versions of the base Stable Diffusion models (10-15% if I recall correctly), but with millions of training images...

**Comment by treksis** [comment]
thank you for the kindly answer. case closed
